go.mod

```diff
@@ -81,7 +81,7 @@ require (
 	golang.org/x/sync v0.0.0-20210220032951-036812b2e83c
 	golang.org/x/term v0.0.0-20210927222741-03fcf44c2211
 	golang.org/x/text v0.3.7
-	golang.org/x/tools v0.1.9
+	golang.org/x/tools v0.1.10
```

Comment:
Hadn't circled back since this wasn't priority, but looked at the diff already here: https://github.com/smartcontractkit/chainlink/pull/6360#issuecomment-1090245057
cc: @smartcontractkit/prodsec-public